models/pinDao.js

```diff
@@ -26,6 +26,27 @@ const getMainPinInfos = async (userId, pageSize, page) => {
     LIMIT ? OFFSET ?;`, [userId, pageSize, page])
 }
 
+const getPinInfo = async (pinId) => {
+    return await appDataSource.query(`
+    SELECT
+        pin.img_url imgUrl,
+        u.nickname,
+        pin.title,
+        pin.contents,
+        (SELECT COUNT(follower.following_id)
+        FROM follower
+        INNER JOIN user
+        ON user.id = follower.follower_id
+        WHERE user.id = u.id) follower
+	FROM pin
+	INNER JOIN	board
+	ON pin.board_id = board.id
+	INNER JOIN user u
+	ON board.user_id = u.id
+	WHERE pin.id = ?;`, [pinId])
```

Comment:
```suggestion
      SELECT
        pin.img_url imgUrl,
        u.nickname,
        pin.title,
        pin.contents,
        (
          SELECT COUNT(follower.following_id)
          FROM follower
          INNER JOIN user
            ON user.id = follower.follower_id
          WHERE user.id = u.id
        ) follower
	FROM pin
	INNER JOIN board
	  ON pin.board_id = board.id
	INNER JOIN user u
	  ON board.user_id = u.id
	WHERE pin.id = ?;`, [pinId])
```
성민님! SQL이 비록 string이지만 가독성을 위해 팀 내 기준을 갖고 indent를 맞춰주는 것이 좋습니다.

Reply:
해당 사항 반영하여 수정완료하였습니다.